ang/test/civicase/CaseOverview.spec.js

```diff
@@ -1,9 +1,16 @@
 /* eslint-env jasmine */
-(function (_) {
+(function ($, _) {
   describe('CaseOverview', function () {
-    var element, $q, $scope, $rootScope, $compile, CasesOverviewStats, crmApi, targetElementScope;
+    var element, $q, $scope, $rootScope, $compile, BrowserCache,
+      CasesOverviewStats, crmApi, targetElementScope;
 
-    beforeEach(module('civicase', 'civicase.data', 'civicase.templates'));
+    beforeEach(module('civicase', 'civicase.data', 'civicase.templates', function ($provide) {
+      BrowserCache = jasmine.createSpyObj('BrowserCache', ['clear', 'get', 'set']);
```

Comment:
Lets create a `BrowserCache` mock so that we dont have to do this every time we use `BrowserCache`.